docs/can-guides/commitment/recipes/file-navigator/file-navigator-advanced.md

```diff
@@ -12,22 +12,24 @@ for an example that doesn't make data requests.
 
 The final widget looks like:
 
-<a class="jsbin-embed" href="https://jsbin.com/diqeyoj/30/embed?js,output">
-  Finished version of the CanJS File Navigator Guide (Advanced) on jsbin.com
-</a>
-<a href="https://jsfiddle.net/donejs/0gxdzLa2/">Open in JSFiddle</a>
+<p class="codepen" data-height="265" data-theme-id="0" data-default-tab="js,result" data-user="cherifGsoul" data-slug-hash="zbxLGW" style="height: 265px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid black; margin: 1em 0; padding: 1em;" data-pen-title="File Navigator advanced [Finished]">
+  <span>See the Pen <a href="https://codepen.io/cherifGsoul/pen/zbxLGW/">
+  File Navigator advanced [Finished]</a> by Mohamed Cherif Bouchelaghem (<a href="https://codepen.io/cherifGsoul">@cherifGsoul</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
 
-> Note: If you don’t see any files show up, run the JS Bin again. This
-> JS Bin uses randomly generated files, so it’s possible nothing shows up.
+> Note: If you don’t see any files show up, run the CodePen again. This
+> CodePen uses randomly generated files, so it’s possible nothing shows up.
 
-__Start this tutorial by cloning the following JS Bin__:
+__Start this tutorial by cloning the following CodePen__:
```

Comment:
This should have the same instructions as the others.

Reply:
Not sure I understand what you mean! Did I change anything else except `JS Bin` to `CodePen`?

Reply:
Yeah, they should all say: **START THIS TUTORIAL BY CLICKING THE “EDIT ON CODEPEN” BUTTON IN THE TOP RIGHT CORNER OF THE FOLLOWING EMBED:**